lib/sycamore/sycamore/query/execution/operations.py

```diff
@@ -263,9 +237,15 @@ def filter_fn_join(doc: Document) -> bool:
     return filter_fn_join
 
 
-def join_operation(docset1: DocSet, docset2: DocSet, field1: str, field2: str) -> DocSet:
+def inner_join_operation(docset1: DocSet, docset2: DocSet, field1: str, field2: str) -> DocSet:
```

Comment:
Should I make inner join, top k, and semantic cluster into DocSet transforms? It isn't hard to do; I wanted to check if is appropriate to move them though

Reply:
Moving to docset makes sense to me